src/Components/Articles.jsx

```diff
@@ -0,0 +1,42 @@
+import { useState, useEffect } from 'react'
+import { fetchAllArticles } from '../api'
+import ArticleCard from './ArticleCard'
+import { Link } from 'react-router-dom'
+
+function Articles() {
+    const [allArticles, setAllArticles] = useState([])
+
```

Comment:
Need some loading state here - right now on a slow connection we don't get any visual feedback as to what's going on as a user